templates/apache-guacamole/0/rancher-compose.yml

```diff
@@ -0,0 +1,70 @@
+version: '2'
+services:
+  guacamole:
+    scale: 1
+    health_check:
+      healthy_threshold: 2
+      response_timeout: 2000
+      port: 8080
+      unhealthy_threshold: 3
+      initializing_timeout: 60000
+      interval: 2000
+      strategy: recreate
+      request_line: GET "/guacamole" "HTTP/1.0"
+      reinitializing_timeout: 60000
+  guacd:
+    scale: 1
+    health_check:
+      healthy_threshold: 2
+      response_timeout: 2000
+      port: 4822
+      unhealthy_threshold: 3
+      initializing_timeout: 60000
+      interval: 2000
+      strategy: recreate
+      reinitializing_timeout: 60000
+  setup-get-db-script:
+    scale: 1
+  db-data:
+    scale: 1
+  database:
+    scale: 1
+    health_check:
+      healthy_threshold: 2
+      response_timeout: 2000
+      port: 3306
+      unhealthy_threshold: 3
+      initializing_timeout: 60000
+      interval: 2000
+      strategy: recreate
+      reinitializing_timeout: 60000
+.catalog:
+  name: "apache-guacamole"
+  version: "0.1-rancher1"
+  description: " Apache Guacamole is a clientless remote desktop gateway. It supports standard protocols like VNC, RDP, and SSH."
+  minimum_rancher_version: v0.46.0
+  questions:
+    - variable: "guacamole_db"
+      label: "MySQL Database Name"
+      description: "Your MYSQL guacamole database name"
+      default: "guacamole_db"
+      required: true
+      type: "string"
+    - variable: "guacamole_user"
+      label: "MySQL Database User"
+      description: "Your MYSQL guacamole user name"
+      required: true
+      default: "guacamole_user"
+      type: "string"
+    - variable: "guacamole_password"
+      label: "MySQL Database Password"
+      description: "Your MYSQL guacamole user password. Defaults to \"guacamole_password\""
+      required: true
+      default: "guacamole_password"
+      type: "password"
+    - variable: "public_port"
```

Comment:
This variable is not referenced anywhere in the catalog.

Reply:
Solved...